include/UnicycleDirectController.h

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright (C) 20022 Fondazione Istituto Italiano di Tecnologia
+ * Authors: Stefano Dafarra
+ * CopyPolicy: Released under the terms of the LGPLv2.1 or later, see LGPL.TXT
+ *
+ */
+
+#ifndef UNICYCLEDIRECTCONTROLLER_H
+#define UNICYCLEDIRECTCONTROLLER_H
+
+#include "UnicycleBaseController.h"
+
+class UnicycleDirectController : public UnicycleBaseController
```

Comment:
If I correctly understand this allows the robot to move in the y direction (i.e. it brakes the holonomic constraint) if this is the case I would avoid calling it `UnicycleDirectController`

Reply:
Nope, this one controller of the unicycle that allows the user to set the unicycle velocities directly. The other controller is the ``PersonFollowing`` controller, where the user sets a desired point to follow.

in order to move sideways I have extended the dynamics of the unicycle in https://github.com/robotology/unicycle-footstep-planner/blob/357445516d1504fe809fdf2ef5696b39beee9c9f/src/ControlledUnicycle.cpp#L55-L56, assuming to have an additional control input. The ``PersonFollowing`` controller simply sets this control input always to zero.